README.md

```diff
@@ -13,39 +14,28 @@
 ## Install
 
 ```sh
-yarn
+pnpm install
 ```
 
-## Build
-
 ```sh
-yarn build
+pnpm dev
 ```
 
-## Lint
+## Build
 
 ```sh
-yarn lint
+pnpm build
 ```
 
-## Examples
+## Lint
 
 ```sh
-yarn eg:gen-evm-addrs
```

Comment:
Add instruction for running the SDK examples. You can add this into `README.md` inside `packages/sdk` folder.

The monorepo `README.md` need not have these sdk examples.

Reply:
the examples right now would not be able to run, because it requires a react native env, if you want to run them, 

You need to import them on the `apps/auto-app` and then execute them on the app, to run a `apps/auto-app` you can run inside that folder `pnpm expo start` this one will generate a QR code. I advice you to install expo.dev in your mobile scan the QR code and the app will run, and you will see the console logs 

if you do have an emulator in your computer you will can just run `pnpm expo android` or `pnpm expo ios`

Reply:
Ok followed.